bokehjs/src/lib/models/text/mathjax/index.ts

```diff
@@ -12,19 +12,26 @@ RegisterHTMLHandler(adaptor)
 
 const svg = new SVG({fontCache: "local"})
 
-const options = {
-  display: true,
-  em: 16,
-  ex: 8,
-  containerWidth: 80*16,
+type ConvertOptions = {
+  display?: boolean
+  em?: number
+  ex?: number
+  containerWidth?: number
 }
 
 type TeXMacros = {[key: string]: string | [string, number]}
 
-export function tex2svg(formula: string, macros: TeXMacros = {}): HTMLElement {
+export function tex2svg(formula: string, options?: ConvertOptions, macros: TeXMacros = {}): HTMLElement {
+  const convert_options: ConvertOptions = {
+    display: true,
+    em: 16,
+    ex: 8,
+    containerWidth: 80*16,
```

Comment:
This should have kept as "defaults" and later on you should use `{...defaults, ...options}`, which states the intent more clearly than this (when seeing this code I thought you would be adding more options, instead of just overriding the defaults).